README.qmd

```diff
@@ -25,83 +75,133 @@ Additionally, a script has been provided to fetch some additional resources for
 
 To get started, we need to:
 
-1.  [Install `nextflow`](https://www.nextflow.io/docs/latest/getstarted.html) system-wide
-
+1.  Install [`nextflow`](https://www.nextflow.io/docs/latest/getstarted.html) system-wide
 2.  Fetch the test data:
 
 ``` bash
 bin/minimal_test.sh
 bin/get_minimal_test_data.sh
 ```
 
-## Running the pipeline
+## Run the Workflow
 
-To actually run the pipeline, we first need to build the components and pipeline:
+The main rnaseq workflow is available via [Viash Hub](https://viash.io/viash-hub/rnaseq.vsh/), where you can receive instructions on how to run the main workflow as well as individual subworkflows or components.
 
-``` bash
-viash ns build --setup cb --parallel
-```
+To run the main workflow, browse to the [rnaseq](https://www.viash-hub.com/packages/rnaseq/v0.2.0/components/workflows/rnaseq) workflow on Viash Hub. Here you can find an overview on the formats of the input and output files, as well as a detailed list of required and optional parameters to run the workflow.
+
+### Run using Nextflow
```

Comment:
Should we specify that this example will only work on Linux and requires some familiarity with the cli?